frontend/src/services/FileHandlerService.ts

```diff
@@ -117,7 +117,7 @@ const uploadFile = async ({
   // POST generated formData to presigned url.
   const response = await postToPresignedUrl(postData.url, formData)
 
-  const encodedFileId = encodeURIComponent(fileId)
+  const encodedFileId = encodeURIComponent(postData.fields.key)
```

Comment:
`encodedFileId` is concatenated straight into a url below, so just to clarify, what format do we expect `postData.fields.key` to have? I'm asking to because we have to be careful that meaningful url characters (e.g. `/`) are not encoded.

Reply:
The `fileId` is the input file ID, but the controller itself returns a new file ID because we randomized the fileID in the controller [here](https://github.com/opengovsg/FormSG/commit/3d1d29979d3f20e69c173c740ccbe6391d4a90ba#diff-d25e48a691b72cce48326009d12cc208e5911e05c08ebcacb1a85651e946eb39), which instituted a change to the file handler in angular and we need need to replicate this behavior in react